mailbox/backup/src/test/java/org/apache/james/mailbox/backup/DefaultMailboxBackupTest.java

```diff
@@ -152,10 +155,46 @@ void doBackupShouldOnlyArchiveTheMailboxOfTheUser() throws Exception {
         }
     }
 
+    @Test
+    void backupEmptyAccountThenRestoringItInUser2AccountShouldCreateNoElements() throws Exception {
```

Comment:
is there a test showing that restoring in a non empty account fails?

Reply:
Nice catch, i added one, and i think it improved the design